src/pattern_library/pages/utility-classes/content.md

```diff
@@ -33,9 +33,21 @@ We use this to make the sidebar content stay on screen when a donor scrolls the
 </div>
 ```
 
+## Stretch Single Content Card
+
+When we have short pages, like a 404 or an error the main content is shorter than the sidebar and looks weird. This makes a content card that's an only child stretch vertically and fill the space.
```

Comment:
What was the rationale for making this a wrapper div and not a data attribute on content card?